codebase/factory/appfactory/setup_rest_server.go

```diff
@@ -16,6 +16,9 @@ func SetupRESTServer(service factory.ServiceFactory, opts ...restserver.OptionFu
 		restserver.SetSharedListener(service.GetConfig().SharedListener),
 		restserver.SetDebugMode(env.BaseEnv().DebugMode),
 		restserver.SetJaegerMaxPacketSize(env.BaseEnv().JaegerMaxPacketSize),
+		restserver.SetTlsOption(env.BaseEnv().UseRESTTls),
```

Comment:
jadi yg disini udah gaperlu, cukup lewat `configs/app_factory.go` dari sisi app-nya saja

(dari sisi app-nya)
```suggestion
		if env.BaseEnv().UseREST {
		apps = append(apps, appfactory.SetupRESTServer(service, restserver.SetTLSConfig(getTLSConfig())))
		...
```